app/helpers/view_helper.rb

```diff
@@ -143,9 +143,7 @@ def show_legacy_courses_table?
 
   def base_errors_hash(provider_code, course)
     {
-      "You must say whether you can sponsor visas" =>
-        provider_recruitment_cycle_visas_path(provider_code, course.recruitment_cycle_year),
-      "Select if you can sponsor visas" =>
+      "Select if visas can be sponsored" =>
```

Comment:
@davidgisbey I don't think the first line is used anywhere, so I've deleted it. Think it was just an older version of the copy?

Reply:
yea looks like old copy to me!